raiden-cli/src/raiden.ts

```diff
@@ -98,6 +98,8 @@ function parseArguments() {
         default: '127.0.0.1:5001',
         desc: 'host:port to bind to and listen for API requests',
       },
+    })
+    .options({
```

Comment:
What is the point of having `options` two times?

Reply:
Type inference. If you try to put everything in a single object,you'll see every member becomes `unknown`